resources/quality/ultimaker_s3/um_s3_bb0.8_pva_0.3mm.inst.cfg

```diff
@@ -14,6 +14,7 @@ weight = -3
 [values]
 brim_replaces_support = False
 cool_fan_enabled = =not (support_enable and (extruder_nr == support_infill_extruder_nr))
+material_print_temperature = =default_material_print_temperature + 5
```

Comment:
```suggestion
material_print_temperature = =default_material_print_temperature - 5
```

Reply:
Before this commit, there were (very grossly) temperature offsets of +10 defined in the BB0.8 variant files (for example `resources/variants/ultimaker_s7_bb0.8.inst.cfg`). These are now removed from there.